File: Backjoon/Backjoon10809/Backjoon10809.c
#include <Stdio.h>

int main(){
    char alpa[26] = {'a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm' ,'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'};
    int num[26];
    char *S = "baekjoon";
    int i;
    for(i = 0; i < 26; i++)    
        num[i] = -1;
    i = -1;

    while (S[++i] != '\0')
        for(int ii = 0; ii < 26; ii++)
            if(alpa[ii] == S[i] && num[ii] == -1)
                num[ii] = i;

    for(i = 0; i < 26; i++)    
        printf("%d ", num[i]);
        
    return 0;
}

File: Baekjoon_step/Backjoon10809/Backjoon10809.py
alpa = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm' ,'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z']
num = []
S = "baekjoon"
for i in range(0, 26): 
        num.append(-1)
i = -1

for i in range(0, len(S)):
    for ii in range(0, 26): 
        if(alpa[ii] == S[i] and num[ii] == -1):
            num[ii] = i

for i in range(0, 26):    
    print(num[i], end=" ")